.github/workflows/bundle-deploy-eas-update.yml

```diff
@@ -49,69 +49,22 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: ⬇️ Get last successful deployment commit from the cache
-        id: get-base-commit
-        uses: actions/cache@v4
-        with:
-          path: last-successful-commit-hash.txt
-          key: last-successful-deployment-commit-${{ github.ref_name }}-${{ github.sha }}
-          restore-keys: |
-            last-successful-deployment-commit-${{ github.ref_name }}-
-
-      - name: Add the last successful deployment commit to the output
-        id: last-successful-commit
-        run: echo base-commit=$(cat last-successful-commit-hash.txt) >> "$GITHUB_OUTPUT"
-
       - name: ⬇️ Fetch commits from base branch
         if: ${{ github.ref != 'refs/heads/main' }}
         run: git fetch origin main:main --depth 100
 
-      # This should get the current production release's commit's hash to see if the update is compatible
-      - name: 🕵️ Get the base commit
-        id: base-commit
-        run: |
-          if ${{ inputs.channel == 'production' }}; then
-            echo base-commit=$(git show-ref -s ${{ inputs.runtimeVersion }}) >> "$GITHUB_OUTPUT"
-          else
-            echo base-commit=${{ steps.last-successful-commit.base-commit }} >> "$GITHUB_OUTPUT"
-          fi
-
-      - name: ✓ Make sure we found a base commit
-        run: |
-          if [ -z "${{ steps.base-commit.outputs.base-commit }}" && ${{ inputs.channel == 'production' }} ]; then
-            echo "Could not find a base commit for this release. Exiting."
-            exit 1
-          fi
-
       - name: 🔧 Setup Node
         uses: actions/setup-node@v4
         with:
           node-version-file: .nvmrc
           cache: yarn
 
-      - name: ⚙️ Install Dependencies
-        run: yarn install
-
```

Comment:
The custom action will handle installing dependencies, since it needs to install deps from both commits. Therefore, we are removing the `yarn install` step from this workflow.

The action will:

1. Checkout the previous commit and install deps
2. Fingerprint that commit
3. Checkout the current commit and install deps
4. Fingerprint the current commit
5. Diff the two fingerprints

Afterwards, we will have the proper deps installed, so there is no need to install them prior to running the fingerprint action.